src/apps/loan-list/modal/renew-loans-modal-content.tsx

```diff
@@ -131,6 +139,24 @@ const RenewLoansModalContent: FC<RenewLoansModalContentProps> = ({
             );
           })}
         </ul>
+        {!topButtonsVisible && (
+          <div
+            className="modal-loan__buttons modal-loan__buttons--bottom"
+            style={{ position: "fixed", left: 0, bottom: 0, right: 0 }}
```

Comment:
Please move styling to the design system or add a comment describing why it is necessary in this case.

Normally we do not use inline styling and based on the current implementation and commit comments I do not see any apparent reason to do so here either.

As far as I can tell it could be implemented by adding a new modifier to the `modal-loan__buttons` element.

Reply:
oh I forgot this, I have actually moved it to the design system already 

https://github.com/danskernesdigitalebibliotek/dpl-design-system/blob/3b1e783a8809165988bbb14b81bc0ff857734872/src/stories/Library/Modals/modal-loan/modal-loan.scss#L53

Good catch, I'll remove it from here.